content/docs/1.5/scalers/cron.md

```diff
@@ -0,0 +1,50 @@
++++
+title = "Cron"
+layout = "scaler"
+availability = "v1.5+"
+maintainer = "Community"
+description = "Scale applications based on a cron schedule."
+go_file = "cron_scaler"
++++
+
+### Trigger Specification
+
+This specification describes the `cron` trigger that scales based on a Cron Schedule.
+
+```yaml
+triggers:
+- type: cron
+  metadata:
+    # Required
+    timezone: Asia/Kolkata  # The acceptable values would be a value from the IANA Time Zone Database.
+    start: 0 0/30 * * *
+    end: 0 15/30 * * *
+    desiredReplicas: "10"
+```
+
+**Parameter list:**
+
+- `timezone` is one of the acceptable values from the IANA Time Zone Database. Required.
```

Comment:
Would be nice to add link to the list of timezones, so KEDA user could find the suitable easily.

Reply:
// LoadLocation returns the Location with the given name.
//
// If the name is "" or "UTC", LoadLocation returns UTC.
// If the name is "Local", LoadLocation returns Local.
//
// Otherwise, the name is taken to be a location name corresponding to a file
// in the IANA Time Zone database, such as "America/New_York".
//
// The time zone database needed by LoadLocation may not be
// present on all systems, especially non-Unix systems.
// LoadLocation looks in the directory or uncompressed zip file
// named by the ZONEINFO environment variable, if any, then looks in
// known installation locations on Unix systems,
// and finally looks in $GOROOT/lib/time/zoneinfo.zip.

Reply:
This is the comment from the code zoneinfo.go
Does it make sense to capture this kind of information in the documentation?

Reply:
@zroubalik have added the link to the list of timezones